test/test.py

```diff
@@ -0,0 +1,243 @@
+#!/usr/bin/python3
+
+#
+# This file is part of the PyRDP project.
+# Copyright (C) 2018, 2020 GoSecure Inc.
+# Licensed under the GPLv3 or later.
+#
+
+from pathlib import Path
+
+from scapy.all import *
+
+from pyrdp.core import Observer
+from pyrdp.enum import PointerFlag
+from pyrdp.logging import LOGGER_NAMES, SessionLogger
+from pyrdp.mitm import MITMConfig, RDPMITM
+from pyrdp.mitm.ClipboardMITM import PassiveClipboardStealer
+from pyrdp.mitm.MITMRecorder import MITMRecorder
+from pyrdp.mitm.state import RDPMITMState
+from pyrdp.pdu import ClipboardPDU, FastPathMouseEvent, FastPathPDU, FormatDataResponsePDU
+
+
+def bytesToIP(data: bytes):
+    return ".".join(str(b) for b in data)
+
+
+def parseExportedPdu(packet: packet.Raw):
```

Comment:
Code duplicated from an active PR. One of us will rebase and refactor to avoid this useless duplication.